bin/setenv-docker-customize.sh

```diff
@@ -389,6 +398,37 @@ else
   add_in_exo_configuration "exo.email.smtp.socketFactory.port="
   add_in_exo_configuration "exo.email.smtp.socketFactory.class="
 
+  # Cluster configuration
+  if [ "${EXO_CLUSTER}" = "true" ]; then
+      add_in_exo_configuration "exo.cluster.node.name=${EXO_CLUSTER_NAME}"
+      JCR_CLUSTER_HOSTS=""
+      IDM_CLUSTER_HOSTS=""
+      COMETD_CLOUD_HOSTS=""
```

Comment:
because of the target variable : exo.cometd.oort.cloud. But you are right, COMETD_CLUSTER_HOSTS is better.